monai/transforms/spatial/array.py

```diff
@@ -1859,6 +1756,7 @@ def __init__(
         padding_mode: str = GridSamplePadMode.BORDER,
         norm_coords: bool = True,
         device: torch.device | None = None,
+        align_corners: bool = True,
```

Comment:
This class seems wrong after change.
If adding a test case with `align_corners  = False`, we are able to see errors in `test_resampler.py`.